diff -Naur mariadb-10.11.17/cmake/pcre.cmake mariadb-10.11.17_patched/cmake/pcre.cmake
--- mariadb-10.11.17/cmake/pcre.cmake	2026-05-08 07:43:18.000000000 +0200
+++ mariadb-10.11.17_patched/cmake/pcre.cmake	2026-05-27 09:24:27.696297932 +0200
@@ -65,7 +65,7 @@
   ExternalProject_Add(
     pcre2
     PREFIX   "${dir}"
-    URL "https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.47/pcre2-10.47.zip"
+    URL "file:///${dir}/pcre2-10.47.zip"
     URL_MD5 7906aec38f872b74f1b925122dde9069
     INSTALL_COMMAND ""
     CMAKE_ARGS
