diff -Naur mariadb-11.8.6/cmake/pcre.cmake mariadb-11.8.6_patched/cmake/pcre.cmake
--- mariadb-11.8.6/cmake/pcre.cmake	2026-01-31 14:27:46.000000000 +0100
+++ mariadb-11.8.6_patched/cmake/pcre.cmake	2026-03-24 15:36:42.555989118 +0100
@@ -65,7 +65,7 @@
   ExternalProject_Add(
     pcre2
     PREFIX   "${dir}"
-    URL "https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.47/pcre2-10.47.zip"
+    URL "file:///${dir}/pcre2-10.47.zip"
     URL_MD5 7906aec38f872b74f1b925122dde9069
     INSTALL_COMMAND ""
     CMAKE_ARGS
