diff -Naur mariadb-11.8.3/cmake/pcre.cmake mariadb-11.8.3_patched/cmake/pcre.cmake
--- mariadb-11.8.3/cmake/pcre.cmake	2025-07-28 21:35:02.000000000 +0200
+++ mariadb-11.8.3_patched/cmake/pcre.cmake	2025-11-26 13:38:36.262560264 +0100
@@ -64,8 +64,8 @@
   ExternalProject_Add(
     pcre2
     PREFIX   "${dir}"
-    URL "https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.45/pcre2-10.45.zip"
-    URL_MD5 873da56c6469ec207ca5c5ae9688b83a
+    URL "file:///${dir}/pcre2-10.46.zip"
+    URL_MD5 261f3f9e14b6ce175103c5118a189faa
     INSTALL_COMMAND ""
     CMAKE_ARGS
       "-DCMAKE_WARN_DEPRECATED=FALSE"
